values.yaml

```diff
@@ -168,8 +168,8 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 2
   targetCPUUtilizationPercentage: 60
-  targetMemoryUtilizationPercentage: 60 # available only on Kubernetes ≥1.23 [required "autoscaling/v2"]
-  behavior: {} # available only on Kubernetes ≥1.23 [required "autoscaling/v2"]
+  targetMemoryUtilizationPercentage: 60  # available only on Kubernetes ≥1.23 [required "autoscaling/v2"]
+  behavior: {}  # available only on Kubernetes ≥1.23 [required "autoscaling/v2"]
```

Comment:
Spurious whitespace changes here. Please remove.

Reply:
Removes spaces.  Yamllint has a [rule](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments:~:text=min%2Dspaces%2Dfrom%2Dcontent%20is%20used%20to%20visually%20separate%20inline%20comments%20from%20content.%20It%20defines%20the%20minimal%20required%20number%20of%20spaces%20between%20a%20comment%20and%20its%20preceding%20content.) that requires two spaces between content and comments by default.

Reply:
Ahh, interesting. I've never seen that rule before. Thanks for the context! I didn't know yamllint existed, and I'm glad it does. CI for this repo is currently doing `helm lint --strict`, which doesn't enforce nearly as much as one might naively expect it to. We should add yamllint to this repo's CI and get a PR together to fix any current violations so we have a basis for code style consistency.